<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Files to be checked in</string>
    </property>
    <widget class="QPushButton" name="checkInButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Check In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browseButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeAllButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove all</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="addFilesListWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>361</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCheck_In">
    <property name="title">
     <string>Check In</string>
    </property>
    <addaction name="actionCheck_In_All_Files"/>
   </widget>
   <widget class="QMenu" name="menuCheck_Out">
    <property name="title">
     <string>Check Out</string>
    </property>
   </widget>
   <addaction name="menuCheck_In"/>
   <addaction name="menuCheck_Out"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCheck_In_All_Files">
   <property name="text">
    <string>Check In All Files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
